src/Plugin.php

```diff
@@ -44,11 +44,21 @@ public static function get_instance() {
 	 */
 	public function init() {
 
-		add_action( 'init', array( $this, 'load_plugin_textdomain' ) );
-		add_action( 'init', array( $this, 'register_admin_area' ) );
-		add_filter( 'plugin_action_links_' . plugin_basename( FILE_UPLOAD_TYPES_PLUGIN_FILE ), array( $this, 'plugin_action_links' ) );
-		add_filter( 'upload_mimes', array( $this, 'allowed_types' ) );
-		add_filter( 'wp_check_filetype_and_ext', array( $this, 'real_file_type' ), 999, 3 );
+		$this->hooks();
+	}
+
+	/**
+	 * Register hooks.
+	 *
+	 * @since {VERSION}
+	 */
+	private function hooks() {
+
+		add_action( 'init', [ $this, 'load_plugin_textdomain' ] );
+		add_action( 'init', [ $this, 'register_admin_area' ] );
```

Comment:
Since this is needed for the admin area, perhaps we could do it on the `admin_init` hook OR with `is_admin()` conditional check. 

Let's add some a `todo` comment for future investigation.

Reply:
Instead `todo` comment, I am creating issue (if we decide to work on it, at some point `todo` comment would be changed into issue so I am just bypassing one step)

https://github.com/awesomemotive/file-upload-types/issues/48